src/components/ec-blog-item/ec-blog-item.vue

```diff
@@ -1,28 +1,34 @@
 <template>
-  <div class="ec-main-container ec-blog-item">
-    <div class="tw-grid tw-border-b tw-border-solid tw-border-gray-6">
+  <transition-group
+    class="ec-main-container ec-blog-item"
```

Comment:
not sure about using these two at the same time, they are not utility classes but component classes based on utility classes, you're mixing responsibilities and if you change one here you might have unintended consequences